src/Solutions/SolutionProviders/BadMethodCallSolutionProvider.php

```diff
@@ -62,18 +61,21 @@ protected function getClassAndMethodFromExceptionMessage(string $message): ?arra
 
     protected function findPossibleMethod(string $class, string $invalidMethodName)
     {
-        return $this->getAvailableMethods($class)
-            ->sortByDesc(function (ReflectionMethod $method) use ($invalidMethodName) {
-                similar_text($invalidMethodName, $method->name, $percentage);
+        $methods = $this->getAvailableMethods($class);
 
-                return $percentage;
-            })->first();
+        usort($methods, static function (ReflectionMethod $method) use ($invalidMethodName): float {
```

Comment:
We don't use static closure or return types for closures. Please remove those here and on all other places.